pkg/helmt/helmt.go

```diff
@@ -130,18 +131,38 @@ func template(name string, chart string, values []string, namespace string, skip
 	return execute("helm", execOpts{}, args...)
 }
 
-func fetch(repository, chart, version string, username, password string) error {
+func fetch(repository, chart, version string, username, password string) (string, error) {
+	d, err := tempDir()
```

Comment:
You could add a defer statement to delete the temp dir.
But I am not sure about this as one could also call it a feature to be able to copy and paste the helm template command and use it with the downloaded file.